Spreadsheets (#1783)
enkidevs/curriculum: excel/beginner/charts/how-to-make-a-chart.md

---
author: Stefan-Stojanovic

levels:
  - beginner

aspects:
  - new
  - workout

type: normal

category: how to

---

# How to make a chart

---
## Content

To create a chart in Excel, firs you need to create a table.

Example table:

![simple](https://img.enkipro.com/8f80a2f2b6efaab06bbd5effc0407aff.png)

Now select the whole table, go to `insert` and select `chart`.

![create-chart](https://img.enkipro.com/579a3b86e540f8449faa977b09ae0ea7.png)

This will give you a suggested chart for your data and an option tab to choose to modify your chart.

![options](https://img.enkipro.com/67b72c0e3a80125f28a1d0687875462f.png)

> After you click on a different chart, the chart is instantly updates. Also, if you change something in the data table the chart will update.

Reviewer: ```suggestion
After you click on a different chart or change something in the data table, the chart instantly updates.

```
